testassets/InteropTestsWebsite/Program.cs

```diff
@@ -49,7 +49,7 @@ public static void Main(string[] args)
                         if (useTls)
                         {
                             var basePath = Path.GetDirectoryName(typeof(Program).Assembly.Location);
-                            var certPath = Path.Combine(basePath, "Certs/server1.pfx");
+                            var certPath = Path.Combine(basePath, @"Certs\server1.pfx");
```

Comment:
Split the string and let Path.Combine choose the separator? Occurs in the client sample as well.